src/vm/wren_debug.c

```diff
@@ -22,16 +36,27 @@ void wrenDebugPrintStackTrace(ObjFiber* fiber)
 
     // Skip over stub functions for calling methods from the C API.
     if (fn->module == NULL) continue;
-    
+
     // The built-in core module has no name. We explicitly omit it from stack
     // traces since we don't want to highlight to a user the implementation
     // detail of what part of the core module is written in C and what is Wren.
     if (fn->module->name == NULL) continue;
-    
+
     // -1 because IP has advanced past the instruction that it just executed.
     int line = fn->debug->sourceLines.data[frame->ip - fn->code.data - 1];
-    fprintf(stderr, "[%s line %d] in %s\n",
+
+    char buffer[1024];
+    snprintf(buffer, sizeof buffer, "[%s line %d] in %s\n",
```

Comment:
Ah, this is the annoying part. Microsoft has really poor support for `snprintf()`. :sob: 

http://stackoverflow.com/questions/2915672/snprintf-and-visual-studio-2010

Libuv has its own emulation of it:

```c
#if defined(_MSC_VER) && _MSC_VER < 1900

#include <stdio.h>
#include <stdarg.h>

/* Emulate snprintf() on MSVC<2015, _snprintf() doesn't zero-terminate the buffer
 * on overflow...
 */
int snprintf(char* buf, size_t len, const char* fmt, ...) {
  int n;
  va_list ap;
  va_start(ap, fmt);

  n = _vscprintf(fmt, ap);
  vsnprintf_s(buf, len, _TRUNCATE, fmt, ap);

  va_end(ap);
  return n;
}

#endif
```

We could probably take that and put it into wren_utils.h/.c. Actually, a cleaner solution might be to add a function in wren_utils that both patches Windows support and handles allocating a buffer of the right size. You do that by calling `snprintf()` twice. The first time it will return the number of bytes it needs to fit the entire string. Then you allocate a buffer of that size and call it again with that buffer.